### ISSUE TITLE
Not working on Linux Mint

### ISSUE THREAD

**Issue by LifeWatcher123** [dynobo/normcap#168]
It seems the project is dependent on the 'gnome-shell'. This makes the program throw an error on Cinnamon-based desktops on Linux Mint.

```
20:53:51 - ERROR   - normcap.utils          - L:104 - Uncaught exception! Quitting NormCap!
Traceback (most recent call last):
  File "/home/linuxuser1/.local/bin/normcap", line 8, in <module>
    sys.exit(main())
  File "/home/linuxuser1/.local/lib/python3.8/site-packages/normcap/app.py", line 56, in main
    logger.debug("System info:\n%s", system_info.to_string())
  File "/home/linuxuser1/.local/lib/python3.8/site-packages/normcap/system_info.py", line 201, in to_string
    gnome_shell_version=gnome_shell_version(),
  File "/home/linuxuser1/.local/lib/python3.8/site-packages/normcap/system_info.py", line 37, in gnome_shell_version
    output_raw = subprocess.check_output(["gnome-shell", "--version"], shell=False)
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python5.8/subprocess.py", line 493, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python5.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 3] No such file or directory: 'gnome-shell'
21:53:51 - ERROR   - normcap.utils          - L:107 - Please open an issue with the output above on https://github.com/dynobo/normcap/issues
```

**Comment by dynobo** [comment]
Hi @LifeWatcher123, thanks for reporting this bug. 
NormCap is not dependent on `gnome-shell`, it just checks if its used (to adjust certain behavior). The error you see clearly indicates, that I haven't made this check robust enough, I'll fix that in the next version. :-)

You are using the latest version `NormCap v0.2.10`, correct?  I'm asking, because I fixed a similar issue recently...

**Comment by LifeWatcher123** [comment]
> You are using the latest version NormCap v0.2.10, correct? I'm asking, because I fixed a similar issue recently...

Yes, I am.

**Comment by dynobo** [comment]
This issue should be fixed in the latest [NormCap v0.3.0b2](https://github.com/dynobo/normcap/releases/tag/v0.3.0b2) release. Want to give it a try?

**Comment by dynobo** [comment]
Please reopen, if problem persists.
